cspell.json

```diff
@@ -1,13 +1,12 @@
 // cSpell Settings
 {
-  // Version of the setting file.  Always 0.1
-  "version": "0.1",
+  "version": "0.2",
   // language - current active spelling language
   "language": "en",
   "dictionaryDefinitions": [
     {
         "name": "chef",
-        "file": "./chef_dictionary.txt",
+        "file": "https://raw.githubusercontent.com/chef/chef_dictionary/main/chef.txt",
```

Comment:
Looks like 0.2 deprecates `"file"` in favor of `"path"`

Reply:
In order to not leave redundant code (that may or may not get exercised) I propose removing the spellcheck tasks that `curl` the `chef_dictionary.txt` and adjusting dependencies and references away from those.

Reply:
@tpowell-progress,

Good catch on the `file` vs `path`. I'll fix that.

I was resultant to make any changes to the `workflow` files. Teams can be very sensitive to that.